lib/cisco_node_utils/cisco_cmn_utils.rb

```diff
@@ -224,5 +224,127 @@ def self.array_to_str(array, sort=true)
       end
       lranges.to_s.gsub('..', '-').delete('[').delete(']').delete(' ')
     end
+
+    # normalize_range_array
+    #
+    # Given a list of ranges, merge any overlapping ranges and normalize the
+    # them as a string that can be used directly on the switch.
+    #
+    # Note: The ranges are converted to ruby ranges for easy merging,
+    # then converted back to a cli-syntax ranges.
```

Comment:
*nit* `converted back to a cli-syntax range` or `converted back to cli-syntax rages`